src/views/Access/Employees.tsx

```diff
@@ -1,5 +1,71 @@
-import { Heading } from '@chakra-ui/react';
+import { Image, Grid, GridItem, Box } from '@chakra-ui/react';
+import Supplier from 'views/Access/Supplier';
+import Waiter from 'views/Access/Waiter';
+import Chef from 'views/Access/Chef';
+import Businessman from 'views/Access/Businessman';
+import Driver from 'views/Access/Driver';
+import supplier from 'assets/images/supplier.png';
+import waiter from 'assets/images/waiter.png';
+import chef from 'assets/images/chef.png';
+import businessman from 'assets/images/businessman.png';
+import driver from 'assets/images/driver.png';
 
-const Employees = () => <Heading>MOCK: Pracownicy</Heading>;
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
+
+const Employees = () => (
```

Comment:
Firstly, I don't understand why you created this whole file. 
It's generally empty and referenced components are also mocks.
We should focus on implementing core functionalities rather than creating mock views. 😉

Reply:
Secondly (for the future), all the routes should be stored in `routing.tsx` as @dominikmlynarczyk made.

Reply:
I wanted to make the routing referencing the view of this particular meni element, depending on the occupation to change. This was the arrangement I made with dominique, so I applied the customization of this under the fastest way possible for me. I just did it as a quick fix. Left for future development possibly